### PR TITLE
Make Civet Python 3 ready and make the sample project usable with Django 1.7 and 1.8

### PR DESCRIPTION
**Pull request by lukhnos**
@richardxia cc @aristretto @tajmorton This PR makes two major updates. First, `2to3` was used to upgrade Civet so that it becomes Python 3.4-compatible. Then, the sample project (`testsite`) is also updated, following @tajmorton's updated instruction in https://github.com/counsyl/civet/pull/6, to make it usable with Django 1.7 and 1.8.

I've manually tested `testsite` and made sure that `manage.py runserver` as well as `manage.py test` can be used with the following combinations on OS X:

* Python 2.7.9 + Django 1.6.11
* Python 2.7.9 + Django 1.7.8
* Python 2.7.9 + Django 1.8.1
* Python 3.4.3 + Django 1.8.1

### CODE REVIEW

File: testsite/README.md [hunk 1]
```diff
@@ -1,9 +1,32 @@
...
+You may also want to install Civet in development mode. Back in this project's
+repository root, run:
+
+    python setup.py develop
+
+Once you have everything, run this to see Civet in action:
+
+    python manage.py runserver
 
+Now, modify any Sass or CoffeeScript file in this project, and observer how
```
Comment (richardxia) [inline testsite/README.md]: `s/observer/observe/`

File: testsite/README.md [hunk 2]
```diff
@@ -1,9 +1,32 @@
...
+the affected Sass/CoffeeScript files get recomplied, with relevant messages
```
Comment (richardxia) [inline testsite/README.md]: `s/recomplied/recompiled/`

**Comment by richardxia** [comment]
Changes LGTM! Thanks for making them!

**Comment by lukhnos** [comment]
Typos fixed. Merging. Thanks for the review!

**Pull request merged**